packages/insomnia-app/app/plugins/context/request.js

```diff
@@ -147,6 +154,82 @@ export function init(
     getAuthentication(): Object {
       return renderedRequest.authentication;
     },
+    getUploadFileName(): string {
+      return renderedRequest.body.fileName || '';
+    },
+    setUploadFileName(fileName: string): boolean {
+      if (!renderedRequest.body || renderedRequest.body.mimeType !== CONTENT_TYPE_FILE) {
+        return false;
+      }
+      renderedRequest.body.fileName = fileName;
+      return true;
```

Comment:
This might be clearer to understand without the negative-or conditional 😄 

How about something like the following?

```suggestion
	  const isFile = renderedRequest.body && renderedRequest.body.mimeType === CONTENT_TYPE_FILE;
	  
	  if (isFile) {
	  	renderedRequest.body.fileName = fileName;
	  }
	  return isFile;
```

Reply:
thanks, fixed.